<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TravelAgencyUI</class>
 <widget class="QMainWindow" name="TravelAgencyUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Up and Away</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Graphic_Elements.qrc">
    <normaloff>:/icons/travelagency.jpg</normaloff>:/icons/travelagency.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_Kunde">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>-1</y>
      <width>531</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Kund*in</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>31</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Cust_Id">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>31</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>61</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Vorname</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Vname">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>61</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>91</y>
       <width>57</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nachname</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Nname">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>91</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>121</y>
       <width>35</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Reisen</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>121</y>
       <width>401</width>
       <height>171</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>133</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
     <column/>
     <column/>
     <column/>
     <item row="0" column="0">
      <property name="text">
       <string>Reise-ID</string>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>Begin der Reise</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>Ende der Reise</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Reisen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>531</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Reise</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>21</y>
       <width>44</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Reise-ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Reise_Id">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>21</y>
       <width>108</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>51</y>
       <width>67</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Buchnungen</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>51</y>
       <width>451</width>
       <height>211</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>112</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <item row="0" column="0">
      <property name="text">
       <string>Buchungstyp</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>Start</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>Ende</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>Preis</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>548</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionEinlesen"/>
    <addaction name="actionSpeichern"/>
   </widget>
   <widget class="QMenu" name="menuKund_innen">
    <property name="title">
     <string>Kund*innen</string>
    </property>
    <addaction name="actionSuchen"/>
   </widget>
   <widget class="QMenu" name="menuMeldungen">
    <property name="title">
     <string>Meldungen</string>
    </property>
    <addaction name="actionFehlerhafte_Buchungen"/>
   </widget>
   <widget class="QMenu" name="menuEinstellungen">
    <property name="title">
     <string>Einstellungen</string>
    </property>
    <addaction name="actionKonsistenzchecks"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuKund_innen"/>
   <addaction name="menuMeldungen"/>
   <addaction name="menuEinstellungen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEinlesen"/>
   <addaction name="actionSuchen"/>
   <addaction name="actionSpeichern"/>
   <addaction name="actionKonsistenzchecks"/>
  </widget>
  <action name="actionEinlesen">
   <property name="icon">
    <iconset resource="Graphic_Elements.qrc">
     <normaloff>:/icons/import-icon.png</normaloff>:/icons/import-icon.png</iconset>
   </property>
   <property name="text">
    <string>Einlesen</string>
   </property>
   <property name="toolTip">
    <string>Datei einlesen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSuchen">
   <property name="icon">
    <iconset resource="Graphic_Elements.qrc">
     <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
   </property>
   <property name="text">
    <string>Suchen</string>
   </property>
   <property name="toolTip">
    <string>Kunde suchen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="icon">
    <iconset resource="Graphic_Elements.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFehlerhafte_Buchungen">
   <property name="icon">
    <iconset resource="Graphic_Elements.qrc">
     <normaloff>:/icons/warning.png</normaloff>:/icons/warning.png</iconset>
   </property>
   <property name="text">
    <string>Fehlerhafte Buchungen</string>
   </property>
  </action>
  <action name="actionKonsistenzchecks">
   <property name="icon">
    <iconset resource="Graphic_Elements.qrc">
     <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>Konsistenzchecks</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Graphic_Elements.qrc"/>
 </resources>
 <connections/>
</ui>
